lib/mockRequest.js

```diff
@@ -191,6 +191,59 @@ function createRequest(options) {
         return Accepts.type(types);
     };
 
+    /**
+     * Check if the given `encoding`s are accepted.
+     *
+     * @param {String} ...encoding
+     * @return {String|Array}
+     * @public
+     */
+
+    mockRequest.acceptsEncodings = function(encodings){
+        if (!Array.isArray(encodings)) {
+            encodings = [].slice.call(arguments);
+        }
+
+        var accept = accepts(mockRequest);
+        return accept.encodings.apply(accept, encodings);
```

Comment:
@AndyOGo could you please explain why do you use here 
```
accept.encodings.apply(accept, encodings);
```
instead of simple function invocation
```
accept.encodings(encodings);
```

Reply:
@eugef I just double checked, the API of `accepts.encoding` supports both, passing encodings as Array and flattened args.
https://github.com/jshttp/accepts/blob/2a6e060aebb52813fdb074e9e7f66da1cfa61902/index.js#L127-L144

I basically copy/pasted the code from `express` directly,  which does the `apply`.
https://github.com/expressjs/express/blob/e1b45ebd050b6f06aa38cda5aaf0c21708b0c71e/lib/request.js#L145-L148

You are right, this `apply` here is unnecessary (as it is in express itself) - I will refactor it.

Reply:
@eugef Done.

Thanks a lot for your accurate review, very appreciated 👍

Reply:
@AndyOGo you are welcome and big thanks for the PR.